server/models/Document.js

```diff
@@ -422,13 +471,17 @@ Document.searchForUser = async (
     ],
   });
 
-  return map(results, (result) => ({
-    ranking: result.searchRanking,
-    context: removeMarkdown(unescape(result.searchContext), {
-      stripHTML: false,
-    }),
-    document: find(documents, { id: result.id }),
-  }));
+  return {
+    data: map(results, (result) => ({
+      ranking: result.searchRanking,
+      context: removeMarkdown(unescape(result.searchContext), {
+        stripHTML: false,
+      }),
+      document: find(documents, { id: result.id }),
+    })),
+    totalCount: count,
+    currentPage: Math.floor(offset / limit),
```

Comment:
I think we can just lose `currentPage` if we're not using it for anything – it's just unused code to maintain 😉

Reply:
sure, np